### PR TITLE
Add option for concatenation character

### LINKED ISSUE #46: Telegraf HTTP JSON receiver combines the measurement and field with an underscore
It seems that when using Telegraf HTTP JSON receiver, the measurement name and field name are combined with an underscore. When configuring Telefgraf to send data as graphite output to `carbon-clickhouse`, they are combined with a dot. The dot is in my opinion also more logical.

I use the http transport as it seems smaller in size when compared to the graphite transport. (especially when using tags a lot)

### LINKED ISSUE #47: InfluxDB format support
Please consider adding support for receiving metrics in the Influx format.
http json format breaks the names of metrics for the case if you use the telegraph to collect metrics from exporters.
For example, for the summary metric type with time quantiles, unreadable labels with dots in the name are obtained.

### PR DESCRIPTION
**Pull request by Hipska**
Fixes #46 
Fixes #47

### CODE REVIEW

File: receiver/receiver.go [hunk 1]
```diff
@@ -80,6 +80,16 @@ func ReadTimeout(seconds uint32) Option {
 	}
 }
 
+// ConcatChar creates option for New contructor
```
Comment (Felixoid) [inline receiver/receiver.go:83]: Typo in `constructor`?

**Review by Felixoid — COMMENTED** [review]
Looks good besides small details!

File: receiver/receiver.go (hunk 1 above)
Comment (Hipska) [reply receiver/receiver.go:83]: Good catch, I just copied from the one above 😄

File: README.md [hunk 2]
```diff
@@ -213,6 +213,8 @@ enabled = false
 drop-future = "0s"
 drop-past = "0s"
 drop-longer-than = 0
+# the character to join telegraf metric and field
+concat = "."
```
[the remark this answers is not in the document]
Comment (Hipska) [reply README.md:217]: Correct, added more details on that.

[the remark this answers is not in the document]
Comment (Felixoid) [reply README.md:217]: Ok, but the value is still `.` :-j

[the remark this answers is not in the document]
Comment (Hipska) [reply README.md:217]: Yes, the example should show the most logical value, no?

[the remark this answers is not in the document]
Comment (Hipska) [reply README.md:217]: Yeah, I don't get it either, he said [here](https://github.com/lomik/carbon-clickhouse/issues/46#issuecomment-533044813) that he wanted to keep Prometheus compatibility 🤷

**Pull request merged**